flow-typed/globals.js

```diff
@@ -0,0 +1 @@
+declare var SULU_ADMIN_BUILD_VERSION: string;
```

Comment:
That folder is kind of reserved for typings that are actually loaded from [flow-typed](https://github.com/flow-typed/flow-typed), so I think we have to find another place for that.

Reply:
as far as i understand, `flow-typed` will store its definitions in `flow-typed/npm/*`. but i am fine with any other path too - what would you suggest?

Reply:
as far as i understand, `flow-typed` will store its definitions in `flow-typed/npm/*`. the flow documentation just states that the `flow-typed` directory is a convention for storing definitions. but i am fine with any other path too - what would you suggest?

Reply:
Just googled, that's the way it is supposed to be, even found a [flow documentation page](https://flow.org/en/docs/libdefs/creation/) about it.